techniques/system/common/1.0/promises.st

```diff
@@ -538,9 +538,9 @@ bundle agent check_log_system
   classes:
 
     !android::
-      "rsyslogd" expression => fileexists("${rsyslog_conffile}");
-      "syslogng" expression => fileexists("${syslogng_conffile}");
-      "syslogd"  expression => fileexists("${syslog_conffile}");
+      "rsyslogd_conffile_present" expression => fileexists("${rsyslog_conffile}"), scope => "namespace";
```

Comment:
I prefer avoiding too generic classes in default namespace, but this can be discussed.